src/core/read_halos-velociraptor.c

```diff
@@ -206,20 +204,18 @@ void read_trees__velociraptor(int snapshot,
       READ_TREE_ENTRY_PROP(Head, long, H5T_NATIVE_LONG);
       READ_TREE_ENTRY_PROP(Tail, long, H5T_NATIVE_LONG);
       READ_TREE_ENTRY_PROP(hostHaloID, long, H5T_NATIVE_LONG);
-      READ_TREE_ENTRY_PROP(Mass_200crit, double, H5T_NATIVE_DOUBLE);
-      READ_TREE_ENTRY_PROP(Mass_FOF, double, H5T_NATIVE_DOUBLE);
-      READ_TREE_ENTRY_PROP(Mass_tot, double, H5T_NATIVE_DOUBLE);
-      READ_TREE_ENTRY_PROP(R_200crit, double, H5T_NATIVE_DOUBLE);
-      READ_TREE_ENTRY_PROP(Vmax, double, H5T_NATIVE_DOUBLE);
-      READ_TREE_ENTRY_PROP(Xc, double, H5T_NATIVE_DOUBLE);
-      READ_TREE_ENTRY_PROP(Yc, double, H5T_NATIVE_DOUBLE);
-      READ_TREE_ENTRY_PROP(Zc, double, H5T_NATIVE_DOUBLE);
-      READ_TREE_ENTRY_PROP(VXc, double, H5T_NATIVE_DOUBLE);
-      READ_TREE_ENTRY_PROP(VYc, double, H5T_NATIVE_DOUBLE);
-      READ_TREE_ENTRY_PROP(VZc, double, H5T_NATIVE_DOUBLE);
-      READ_TREE_ENTRY_PROP(Lx, double, H5T_NATIVE_DOUBLE);
-      READ_TREE_ENTRY_PROP(Ly, double, H5T_NATIVE_DOUBLE);
-      READ_TREE_ENTRY_PROP(Lz, double, H5T_NATIVE_DOUBLE);
+      READ_TREE_ENTRY_PROP(Mass_200crit, float, H5T_NATIVE_FLOAT);
+      READ_TREE_ENTRY_PROP(Mass_FOF, float, H5T_NATIVE_FLOAT);
+      READ_TREE_ENTRY_PROP(Mass_tot, float, H5T_NATIVE_FLOAT);
+      READ_TREE_ENTRY_PROP(R_200crit, float, H5T_NATIVE_FLOAT);
+      READ_TREE_ENTRY_PROP(Vmax, float, H5T_NATIVE_FLOAT); // WHAT ARE THE UNITS OF THIS Vmax?? It's never converted with little_h!
```

Comment:
velocity normally doesn't have little_h in it

Reply:
but it's better check with chris

Reply:
and vmax doesn't normally have be scaled by redshift. but again check with chris plz!

Reply:
Chris just confirmed this so all good!